test/milton/MiltonEvents.t.sol

```diff
@@ -0,0 +1,526 @@
+            address(miltonUsdt),
+            address(miltonStorageUsdt),
+            address(stanleyUsdt)
+        );
+        prepareApproveForUsersUsd(_users, usdtMockedToken, address(josephUsdt), address(miltonUsdt));
+        prepareMilton(miltonUsdt, address(josephUsdt), address(stanleyUsdt));
+        prepareJoseph(josephUsdt);
+        prepareIpToken(ipTokenUsdt, address(josephUsdt));
+        // when
+        _miltonSpreadModel.setCalculateQuotePayFixed(TestConstants.PERCENTAGE_6_18DEC); // 6%
+        vm.prank(_userOne);
+        iporOracle.itfUpdateIndex(address(usdtMockedToken), TestConstants.PERCENTAGE_5_18DEC, block.timestamp); // 5%, PERCENTAGE_5_18DEC
+        vm.prank(_liquidityProvider);
+        josephUsdt.itfProvideLiquidity(TestConstants.USD_28_000_6DEC, block.timestamp); // USD_28_000_6DEC
+        vm.prank(_userTwo);
+        miltonUsdt.itfOpenSwapPayFixed(
+            block.timestamp, // openTimestamp
+            TestConstants.USD_10_000_6DEC, // totalAmount, USD_10_000_6DEC
+            TestConstants.PERCENTAGE_6_18DEC, // acceptableFixedInterestRate, 6%
+            TestConstants.TC_IPOR_PUBLICATION_AMOUNT_18DEC // leverage, LEVERAGE_18DEC
+        );
+        vm.prank(_userOne);
+        iporOracle.itfUpdateIndex(address(usdtMockedToken), TestConstants.PERCENTAGE_160_18DEC, block.timestamp); // PERCENTAGE_160_18DEC
+        vm.prank(_userThree);
+        vm.expectEmit(true, true, false, false);
+        emit CloseSwap(
+            1, // swapId
+            address(usdtMockedToken), // asset
+            block.timestamp + TestConstants.SWAP_DEFAULT_PERIOD_IN_SECONDS, // closeTimestamp, 28 days, PERIOD_28_DAYS_IN_SECONDS
+            _userThree, // liquidator
+            18957318804000000000000, // transferredToBuyer
+            TestConstants.TC_LIQUIDATION_DEPOSIT_AMOUNT_18DEC, // transferredToLiquidator
+            TestConstants.TC_INCOME_TAX_18DEC // incomeFeeValue
+        );
+        miltonUsdt.itfCloseSwapPayFixed(
+            1, // swapId
+            block.timestamp + TestConstants.SWAP_DEFAULT_PERIOD_IN_SECONDS // closeTimestamp, 28 days, PERIOD_28_DAYS_IN_SECONDS
+        );
+    }
+
+    function testShouldEmitMiltonSpreadModelChanged() public {
+        // given
+        MockTestnetToken daiMockedToken = getTokenDai();
+        ItfIporOracle iporOracle = getIporOracleAsset(_userOne, address(daiMockedToken), 0);
+        MockCase0Stanley stanleyDai = getMockCase0Stanley(address(daiMockedToken));
+        MiltonStorage miltonStorageDai = getMiltonStorage();
+        ItfMiltonDai miltonDai = getItfMiltonDai(
+            address(daiMockedToken),
+            address(iporOracle),
+            address(miltonStorageDai),
+            address(_miltonSpreadModel),
+            address(stanleyDai)
+        );
+        address oldMiltonSpreadModel = miltonDai.getMiltonSpreadModel();
+        address newMiltonSpreadModel = address(_userThree);
+        // when
+        vm.expectEmit(true, true, true, false);
```

Comment:
Why do you use vm.expectEmit(true, true, true, false) and not vm.expectEmit(true, true, true, true)